src/sentry/models/group.py

```diff
@@ -67,8 +67,7 @@
 
 logger = logging.getLogger(__name__)
 
-_short_id_re = re.compile(r"^(.*?)(?:[\s_-])([A-Za-z0-9]+)$")
-
+_short_id_re = re.compile(r"^(?:[^:]+:)?(.*?)(?:[\s_-])([A-Za-z0-9]+)$")
```

Comment:
Should this explicitly look for either `<short_id>` or `issue:<short_id>`? Or is the `issue:` already stripped off here?

Reply:
@wedamija yea, probably actually. Was thinking maybe people were doing `issue_id` or something as well an needed to capture that but probably not.